met-web/src/components/engagement/view/SurveyBlock.tsx

```diff
@@ -58,7 +58,7 @@ const SurveyBlock = ({ startSurvey }: SurveyBlockProps) => {
                                 data-testid="SurveyBlock/view-feedback-button"
                                 disabled={!surveyId}
                                 onClick={() => {
-                                    navigate(`/engagements/${savedEngagement.id}/dashboard`);
+                                    navigate(`/engagements/${savedEngagement.id}/dashboard/public`);
```

Comment:
![image](https://github.com/bcgov/met-public/assets/91914654/ddf476c5-2c08-4013-b0ef-5f0cf9e4e5e7)
Can we use this